client/src/Features/MicrosoftLearn/MicrosoftLearnFilterCore.ts

```diff
@@ -44,7 +44,7 @@ export function getFiltersToDisplay(catalog: Catalog | null, filteredContent: Le
   }
 }
 
-function getParentProduct(products: Map<string, Product> | undefined) {
```

Comment:
Why did we need this?

Reply:
I needed to use the function on the UI side, so exported it from here itself.